tool/debian/build.sh

```diff
@@ -86,26 +86,22 @@ if [ -n "$BUILD_ID" ]; then
 fi
 
 FOLDER=$FOLDER_PREFIX-cpp
-if [ $CUDA = "ON" ]; then
-  FOLDER=$FOLDER_PREFIX-cuda$CUDA_VERSION-cudnn$CUDNN_VERSION
-fi
-
 if [ $PYTHON = "ON" ]
 then
   FOLDER=$FOLDER/python-singa
```

Comment:
1. singa-cpp folder has two subfolders： python-singa and singa
2. singa-cuda8.0-cudnn5 does not have the same subfolder.
should we make them consistent by following either 1 or 2?